tests/testthat/test-utils-format.R

```diff
@@ -0,0 +1,18 @@
+context("utils-format")
+
+test_that("pluralise works correctly", {
+  expect_identical(pluralise("[an] index(es)", c("x")), "an index")
+  expect_identical(pluralise("[an] index(es)", c("x", "y")), "indexes")
+})
+
+test_that("pluralise leaves alone parentheses / square brackets that have spaces inside", {
+  expect_identical(pluralise("[an] index(es )", c("x")), "an index(es )")
+  expect_identical(pluralise("[an ] index(es)", c("x", "y")), "[an ] indexes")
+})
```

Comment:
I added a test about leaving parens/brackets with spaces alone. I didn't understand the use case, so the test examples may be dumb. I hope I got your ideas right.